simphony/cuds/abc_modeling_engine.py

```diff
@@ -0,0 +1,213 @@
+
+        Returns
+        ----------
+        ABCParticleContainer
+            A particle container to be used to update/query the internal
+            representation stored inside the modeling-engine. See
+            get_particle_container for more information.
+
+        """
+        pass
+
+    @abstractmethod
+    def delete_lattice(self, name):
+        """Delete a lattice
+
+        Parameters
+        ----------
+        name: str
+            name of lattice to be deleted
+
+        """
+        pass
+
+    @abstractmethod
+    def delete_mesh(self, name):
+        """Delete a mesh
+
+        Parameters
+        ----------
+        name: str
+            name of mesh to be deleted
+
+        """
+        pass
+
+    @abstractmethod
+    def delete_particle_container(self, name):
+        """Delete a particle container
+
+        Parameters
+        ----------
+        name: str
+            name of particle container to be deleted
+
+        """
+        pass
+
+    @abstractmethod
+    def get_lattice(self, name):
+        """ Get lattice
+
+        The returned lattice can be used to query and update the state of the
+        lattice inside the modeling engine.
+
+        Returns
+        ----------
+        ABCLattice
+
+        """
+        pass
+
+    @abstractmethod
+    def get_mesh(self, name):
+        """ Get mesh
+
+        The returned mesh can be used to query and update the state of the
+        mesh inside the modeling engine.
+
+        Returns
+        ----------
+        ABCMesh
+
+        """
+        pass
+
+    @abstractmethod
+    def get_particle_container(self, name):
+        """ Get particle container
+
+        The returned particle container can be used to query and update the
+        state of the particle container inside the modeling engine.
+
+        Returns
+        ----------
+        ABCParticleContainer
+
+        """
+        pass
+
+    @abstractmethod
+    def iter_lattices(self, names=None):
+        """ Returns an iterator over a subset or all of the lattices.
+
+        See get_lattice for more information.
+
+        Parameters
+        ----------
+        names : sequence of str, optional
+            names of specific lattices to be iterated over. If names is not
+            given, then all lattices will be iterated over.
+
+        """
+        pass
+
+    @abstractmethod
+    def iter_meshes(self, names=None):
+        """ Returns an iterator over a subset or all of the meshes.
+
+        See get_mesh for more information.
+
+        Parameters
+        ----------
+        names : sequence of str, optional
+            names of specific meshes to be iterated over. If names is not
+            given, then all meshes will be iterated over.
+
+        """
+        pass
+
+    @abstractmethod
+    def iter_particle_containers(self, names=None):
+        """ Returns an iterator over a subset or all of the particle containers.
+
+        See get_particle_container for more information.
```

Comment:
I do not think that his give more info, I would suggest that we remove such comments